tests/integration/utils.py

```diff
@@ -42,6 +44,30 @@ def run_cmd_on_image(self, cmd: List, tmpdir: Path) -> Tuple[str, int]:
         image_cmd = ["podman", "run", "--rm", "-v", f"{tmpdir}:{tmpdir}:z", self.repository] + cmd
         return run_cmd(image_cmd)
 
+    def delete_image(self):
+        image_cmd = ["podman", "rmi", "--force", self.repository]
+        (output, rc) = run_cmd(image_cmd)
+        if rc != 0:
+            raise RuntimeError(f"Image deletion failed. Output:{output}")
+
+
+def build_image(tmpdir: Path, container_folder: str, test_case: str) -> ContainerImage:
+    image_cmd = [
+        "podman",
+        "build",
+        container_folder,
+        "-v",
+        f"{tmpdir}:/tmp:z",
+        "--network",
+        "none",
+        "--tag",
+        test_case,
+    ]
```

Comment:
Should we build with `--no-cache` to avoid succeeding accidentally?

Reply:
yes, added